libcore/FileOperations/CopyMoveJob.vala

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2022 elementary LLC (https://elementary.io)
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, Inc.,; either version 2 of
+ * the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301, USA.
+ */
+
+public class Files.FileOperations.CopyMoveJob : CommonJob {
+    protected bool is_move = false;
+    protected GLib.List<GLib.File> files;
+    protected GLib.File? destination;
+    protected GLib.HashTable<GLib.File,bool> debuting_files = new GLib.HashTable<GLib.File,bool> (GLib.File.hash, GLib.File.equal);
```

Comment:
I do not think `debuting_files` is actually used anymore so maybe take the opportunity to drop it?